metadata-ingestion/src/datahub/ingestion/source/powerbi/rest_api_wrapper/data_classes.py

```diff
@@ -247,6 +247,7 @@ class Dashboard:
     tiles: List["Tile"]
     users: List["User"]
     tags: List[str]
+    usageMetrics: Optional[Dict[str, Dict]] = None
```

Comment:
please add a stronger type here - what is the inner dict? maybe you should use a dataclass